src/components/Room.tsx

```diff
@@ -58,12 +58,27 @@ function Room() {
     console.log("roomId:", window.location.href.split("/")[3]);
   }, [isConnected]);
 
+  async function shareRoom() {
```

Comment:
I just noticed that this function is declared in `Room.tsx`. A better place for this code to live in would be in `/src/helpers/shareRoom.ts`.

With react, we want to keep the component (`Room` in this case) as clean as possible, so any helper functions need to be moved out of the component.